app-backend/http4s-util/src/main/scala/com/rasterfoundry/http4s/xray/UdpClient.scala

```diff
@@ -0,0 +1,28 @@
+package com.rasterfoundry.http4s.xray
+
+import java.net._
+
+import cats.effect.Sync
+import io.circe.syntax._
+import io.circe.Printer
+
+class UdpClient[F[_]: Sync] {
+
+  val printer = Printer(
+    preserveOrder = true,
+    dropNullValues = true,
```

Comment:
Using a custom `json` printer from circe in order to drop `null`/`None` values to keep Xray happy